middleware/getUsername.js

```diff
@@ -0,0 +1,9 @@
+const jwtDecode = require("jwt-decode");
+
+const getUsername = (request, response, next) => {
+  const token = request.cookies.jwt;
+  const { username } = jwtDecode(token);
+  response.json({ username });
```

Comment:
What happens if the user isn't logged in?

Reply:
My understanding is that as I'm using `passport.authenticate()`, this will response with a `401 Unauthorized` status, and the next middleware (getUsername) will not be called